src/colmap/scene/reconstruction.h

```diff
@@ -102,7 +102,9 @@ class Reconstruction {
   // might be taken by the same camera.
   void AddCamera(struct Camera camera);
 
-  // Add new image.
```

Comment:
We should also add comments at the two reconstruction IO functions like: https://github.com/colmap/colmap/pull/2702/files#diff-6ff1c633d8187e2cb2de82d2b9217f65046013c4edfd3c8eaf53e0a95b91461cR40.

Reply:
Done